v2/cacheutils/cache_test.go

```diff
@@ -255,9 +255,6 @@ func TestGenerateFingerPrint(t *testing.T) {
 			assert.Nil(t, err)
 			testFingerPrint, err := GenerateFingerPrint(test.cacheKey)
 			assert.Nil(t, err)
-			fmt.Println(test.name)
-			fmt.Println(fingerPrint)
-			fmt.Println(testFingerPrint)
 			assert.Equal(t, fingerPrint == testFingerPrint, test.wantEqual)
```

Comment:
Can we assert the hash key value in unit test, so that we can detect future regressions if ever the format changed?

Reply:
done